bigml/src/client.rs

```diff
@@ -233,6 +233,9 @@ impl Client {
     }
 
     /// Poll an existing resource, returning it once it's ready.
+    ///
+    /// If an underlying BigML error occurs, it will be returned as
```

Comment:
Does this mean we won't be able to see the error, or just that it will be wrapped in `WaitFailed`?

Reply:
This means that the `Client::wait` and `Client::wait_opt` APIs are now promising to return an `Error::WaitFailed` if the underlying BigML resource enters an error state, and they're promising _not_ to wrap it some higher-level error that the caller would need to pick apart recursively to find the `Error::WaitFailed` with the actual information that's needed.

Reply:
Okay, that's good! So long as the caller still has visibility into what exactly  happened, this looks great!